code/controllers/subsystem/tts.dm

```diff
@@ -123,6 +123,12 @@ SUBSYSTEM_DEF(tts)
 	request.begin_async()
 	in_process_tts_messages += list(entry)
 
+/datum/controller/subsystem/tts/Shutdown()
+	tts_enabled = FALSE
+	for(var/list/data in in_process_tts_messages)
```

Comment:
Can there be any more messages sent, that could cause people to hold up the round just by talking?

Reply:
Also we should make a comment here that this is here to fix a rust-g bug--rust-g itself needs to be able to handle this. DM code shouldn't be able to cause this kind of behavior.

Reply:
> Can there be any more messages sent, that could cause people to hold up the round just by talking?

No, this sets tts_enabled to FALSE so new requests won't get recorded. Additionally, subsystems stop firing so queued messages won't get sent.

> Also we should make a comment here that this is here to fix a rust-g bug--rust-g itself needs to be able to handle this. DM code shouldn't be able to cause this kind of behavior.

Sure